src/Helpers/OidStripper.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\PhpSnmp\Helpers;
+
+use SimPod\PhpSnmp\Transport\SnmpClient;
+use function array_key_first;
+use function assert;
+use function is_string;
+use function Safe\substr;
+use function strlen;
+use function strrpos;
+
+final class OidStripper
+{
+    /**
+     * @param array<string, mixed> $raw
+     *
+     * @return array<int, mixed>
+     */
+    public static function stripLeafOidsParentOid(array $raw) : array
```

Comment:
can you explain the name?

Reply:
parent OID (the one you request): .1.2.3
leaf OID (one that has no children): .1.2.3.4

to strip leafOid parentOid means to strip .1.2.3 part of .1.2.3.4 to get just the "index" 4 as a key

Reply:
stripParentOidFromLeafOids ?

Reply:
it's already long as it is, and the meaning is the same

Reply:
the current name is nonsense tbh..

Reply:
What about just `OidStripper::leafOids`? With the method signature, it makes sense...

Reply:
that implies stripping leaf oids tho. we're stripping parent oid 🤔

Reply:
nah, it's just oid stripper acting on leafOids ... same with `OidStripper::walk`

Reply:
acting on leaf oids would probably look like `oidstripper::strip($dataWithLeafOids)` returning data without oids. 

To me as a user who did not write the code the naming of public api does not help at all in understanding what the class does. Seems like only because we're trying to save 4 symbols in method name 🤔 https://github.com/simPod/PHP-SNMP/pull/7#discussion_r435775796

Reply:
Yeah, I don't want methods with 50 chars names :D Sure, that example makes sense, I would even make it `OidStripper::stripParent($leafData)` ... not sure about the variable, but yeah ... makes sense like that?

Reply:
strip parent is k

btw 50 char names is ok, we don't have limited amount of keystrokes :P

Reply:
Agree to disagree :) alright, will change it to stripParent

Reply:
_I'll answer using golang naming convention_ 

y h m to l p 😱